documentation/book/assembly-upgrade-cluster-operator.adoc

```diff
@@ -0,0 +1,16 @@
+// This assembly is included in the following assemblies:
+//
+// master.adoc
+
+[id='assembly-upgrade-cluster-operator-{context}']
+= Cluster Operator upgrade
+
+Upgrade your Cluster Operator deployment to use a new version of {ProductName}. Supported upgrades are outlined in this section.
+
+The availability of Kafka clusters managed by the Cluster Operator is not affected by the upgrade operation.
+
+ifdef::StrimziUpgrades[]
+include::proc-upgrading-the-cluster-operator-0-10-0-to-0-11-0.adoc[leveloffset=+1]
+endif::StrimziUpgrades[]
+
+include::proc-upgrading-the-cluster-operator-0-9-0-to-0-10-0.adoc[leveloffset=+1]
```

Comment:
Why are the 0.9.0 → 0.10.0 upgrade instructions after the 0.10.1 → 0.11.0 upgrade instructions? It makes it harder to understand then reading through.

Reply:
I assumed because we wanted the later versions first, stacked on top. I've switched around.